processing/src/main/java/io/druid/query/DefaultQueryMetrics.java

```diff
@@ -118,7 +118,9 @@ public void duration(QueryType query)
   @Override
   public void queryId(QueryType query)
   {
+    //CHECKSTYLE.OFF: Regexp
```

Comment:
Please leave a comment here, that this nullToEmpty() is irrelevant to null handling in the data.

Reply:
done.